source/codegen/metadata/niscope/enums.py

```diff
@@ -680,186 +565,127 @@
                 'name': 'NISCOPE_VAL_READY_FOR_REF_EVENT',
                 'value': 10
             },
-            {
-                'name': 'NISCOPE_VAL_REF_CLOCK',
-                'value': 100
-            },
             {
                 'name': 'NISCOPE_VAL_5V_OUT',
-                'python_name': 'FIVE_V_OUT',
```

Comment:
@reckenro are these python names in enums needed? There's a few places I see them and I don't know if they are needed and I need to make more hapigen changes or if I can leave them out